contracts/core/chainlink-oracle/v1/ChainlinkOracle.sol

```diff
@@ -49,53 +43,30 @@ contract ChainlinkOracle is Ownable {
         return price;
     }
 
-
-    /**
-        @dev   Updates aggregator address. Only the contract Owner can call this function.
-        @dev   It emits a `ChangeAggregatorFeed` event.
-        @param aggregator Address of Chainlink aggregator.
-    */
-    function changeAggregator(address aggregator) external onlyOwner {
-        require(aggregator != address(0), "CO:ZERO_AGGREGATOR_ADDR");
-        emit ChangeAggregatorFeed(aggregator, address(priceFeed));
-        priceFeed = IChainlinkAggregatorV3(aggregator);
+    function changeAggregator(address newAggregator) external override onlyOwner {
```

Comment:
I hate to break it but it will reduce the code readability a lot if we keep every doc in the interfaces.

Reply:
I think that if we make a note of this in the README it will be fine. I prefer moving the comments out into the interface, makes the code neater and smaller, and if a dev needs to refer to the NatSpec they can just search the function name and it'll show up.

Reply:
I have/had similar feelings, but on the flipside, those who only import our interfaces will see the docs. If we put the docs in the implementations only, then interfaces will lack any info for importers, who probably need it more (they don't know the system as well as us). What we could do is some CI task or script to copy docs from each to their couterpart.